mappers/bluetooth_mapper/README.md

```diff
@@ -0,0 +1,318 @@
+# Bluetooth Mapper
+
+
+## Introduction
+
+Mapper is an application that is used to connect and control devices. This is an implementation of mapper for 
+bluetooth protocol. The aim is to create an application through which users can easily operate devices using bluetooth protocol for communication to the KubeEdge platform. The user is required to provide the mapper with the information required to control their device through the configuration file these can be changed at runtime by providing the input through the MQTT broker.
+
+## Running the mapper
+
+While running the mapper please keep in mind the following points :-
+
+   1. Please run the binary as root or with root permissions.
+   2. Please ensure that bluetooth service of your device is ON
+   3. Please remember to give one of the following command line parameters for the logging service:
+    
+     -logtostderr=true      	   // Logs are written to standard error instead of to files.
+     -alsologtostderr=true         // Logs are written to standard error as well as to files.
```

Comment:
better to add indent between -alsologtostderr=true and the comments
https://github.com/kubeedge/kubeedge/blob/2694516a98328b2993a05aab09778ebb3e78c08f/mappers/bluetooth_mapper/README.md#running-the-mapper

Reply:
done, thanks